javatests/arcs/showcase/imports/ImportsTest.kt

```diff
@@ -0,0 +1,36 @@
+package arcs.showcase.imports
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import arcs.core.host.toRegistration
+import arcs.showcase.ShowcaseEnvironment
+import arcs.showcase.imports.particles.AcceptImports
+import arcs.showcase.imports.particles.IngestDock
+import arcs.showcase.imports.recipes.GatherImportsPlan
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+class ImportsTest {
+
+    @get:Rule
+    val env = ShowcaseEnvironment(
+        ::IngestDock.toRegistration(),
+        ::AcceptImports.toRegistration()
+    )
+
+    @Test
+    fun teaShop_SupplyChain() = runBlocking {
```

Comment:
Nit: lowercase `_supplyChain`

Although it's probably better to be more descriptive in the method name about what the point of the test actually is.